src/main/java/com/mastercard/developer/utils/EncryptionUtils.java

```diff
@@ -32,16 +31,29 @@ private EncryptionUtils() {
     /**
      * Populate a X509 encryption certificate object with the certificate data at the given file path.
      */
-    public static Certificate loadEncryptionCertificate(String certificatePath) throws CertificateException, FileNotFoundException {
+    public static Certificate loadEncryptionCertificate(String certificatePath) throws CertificateException, IOException {
+        return loadEncryptionCertificate(Files.readAllBytes(Paths.get(certificatePath)));
+    }
+
+    /**
+     * Populate a X509 encryption certificate object with the certificate data at the given certificate data in bytes.
+     */
+    public static Certificate loadEncryptionCertificate(byte[] certificateBytes) throws CertificateException {
         CertificateFactory factory = CertificateFactory.getInstance("X.509");
-        return factory.generateCertificate(new FileInputStream(certificatePath));
+        return factory.generateCertificate(new ByteArrayInputStream(certificateBytes));
     }
 
     /**
      * Load a RSA decryption key from a file (PEM or DER).
      */
     public static PrivateKey loadDecryptionKey(String keyFilePath) throws GeneralSecurityException, IOException {
-        byte[] keyDataBytes = Files.readAllBytes(Paths.get(keyFilePath));
+        return loadDecryptionKey(Files.readAllBytes(Paths.get(keyFilePath)));
+    }
+
+    /**
+     * Load a RSA decryption key from key data in bytes.
+     */
+    public static PrivateKey loadDecryptionKey(byte[] keyDataBytes) throws GeneralSecurityException {
```

Comment:
Don't forget to add tests to the `EncryptionUtilsTest` class, all public methods should be tested.